docs/leap_sapi/sapi_rest.rst

```diff
@@ -145,6 +145,71 @@ explained in the :ref:`sapi_rest_url_token_setup` subsection.
     :ref:`error responses <sapi_rest_general_response_codes>`, SAPI may return
     particular error codes.
 
+.. _leap_sapi_deprecation_message:
+
+SAPI Deprecation Messages
+-------------------------
+
+The Leap service sends deprecation messages when you use deprecated features;
+for example, a message is sent when you use a deprecated solver parameter or
+submit a problem to a deprecated solver.
+
+Deprecation messages are sent in SAPI responses via an HTTP ``X-Deprecation``
+response header, which is an Item Structured Header field
+(`RFC 9651 <https://www.rfc-editor.org/rfc/rfc9651>`_). Thus, a deprecation
+message is a
+List (`RFC 9651, 3.1 <https://www.rfc-editor.org/rfc/rfc9651#name-lists>`_)
+of Token
+(`RFC 9651, 3.3.4 <https://www.rfc-editor.org/rfc/rfc9651#name-tokens>`_)
+items, where each item is associated with the following
```

Comment:
```suggestion
of Tokens
(`RFC 9651, 3.3.4 <https://www.rfc-editor.org/rfc/rfc9651#name-tokens>`_),
each associated with the following
```

To avoid having "Item (RFC, section)"